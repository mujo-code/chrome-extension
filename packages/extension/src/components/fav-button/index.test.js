import React from 'react'
import { create } from 'react-test-renderer'
import { FavButton } from '.'

test('FavButton component matches snapshot', () => {
  const tree = create(
    <FavButton url="foo" title="bar" disable={false} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
