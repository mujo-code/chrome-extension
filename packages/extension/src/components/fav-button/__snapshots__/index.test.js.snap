// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavButton component matches snapshot 1`] = `
<a
  backgroundColor="outerSpace"
  className="css-51z5ye"
  disable={false}
  disabled={true}
  href="foo"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <link
    href="foo"
    rel="preconnect"
  />
  <img
    alt="bar"
    className="css-1numroz"
    height="m"
    onError={[Function]}
    onLoad={[Function]}
    src="https://getmujo.com/api/icon?site=foo"
  />
  <div
    className="css-79elbk"
  >
    <div
      className="css-4pw2fe"
    >
      <span
        className="css-xrmt9z"
      >
        <div
          className="css-nxex8j"
        />
        <div
          className="css-91tplw"
        >
          bar
        </div>
      </span>
    </div>
  </div>
</a>
`;
